pkgs/development/haskell-modules/make-package-set.nix

```
@@ -275,13 +276,12 @@ in package-set { inherit pkgs lib callPackage; } self // {
     # To reload the Hoogle server automatically on .cabal file changes try
```

```suggestion
    # Example usage:
    #  $ nix-shell -p 'haskellPackages.hoogleWithPackages (p: [ p.mtl p.lens ])'
    #
    # To reload the Hoogle server automatically on .cabal file changes try
```

I think it would be nice to have a little comment here so someone reading through this would have an idea of how `hoogleWithPackages` it could be used.